ui/de-lib/src/main/java/org/iplantc/de/admin/desktop/client/toolAdmin/view/ToolAdminDetailsWindow.ui.xml

```diff
@@ -0,0 +1,87 @@
+                                        allowBlank="false"
+                                        ui:field="nameEditor"/>
+                    </form:widget>
+                </form:FieldLabel>
+            </container:child>
+            <container:child layoutData="{layoutData}">
+                <form:FieldLabel text="{appearance.toolImportDescriptionLabel}">
+                    <form:widget>
+                        <form:TextArea readOnly="false"
+                                       enabled="true"
+                                       ui:field="descriptionEditor"/>
+                    </form:widget>
+                </form:FieldLabel>
+            </container:child>
+            <container:child layoutData="{layoutData}">
+                <form:FieldLabel ui:field="typeLabel">
+                    <form:widget>
+                        <form:TextField readOnly="true"
+                                        enabled="false"
+                                        ui:field="typeEditor"/>
+                    </form:widget>
+                </form:FieldLabel>
+            </container:child>
+            <container:child layoutData="{layoutData}">
+                <form:FieldLabel text="{appearance.toolImportAttributionLabel}">
+                    <form:widget>
+                        <form:TextField readOnly="false"
+                                        enabled="true"
+                                        ui:field="attributionEditor"/>
+                    </form:widget>
+                </form:FieldLabel>
+            </container:child>
+            <container:child layoutData="{layoutData}">
+                <form:FieldLabel text="{appearance.toolImportVersionLabel}">
+                    <form:widget>
+                        <form:TextField readOnly="false"
+                                        enabled="true"
+                                        ui:field="versionEditor"/>
+                    </form:widget>
+                </form:FieldLabel>
+            </container:child>
+            <container:child layoutData="{layoutData}">
+                <form:FieldLabel ui:field="locationLabel">
+                    <form:widget>
+                        <form:TextField readOnly="false"
+                                        enabled="true"
+                                        allowBlank="false"
+                                        ui:field="locationEditor"/>
+                    </form:widget>
+                </form:FieldLabel>
+            </container:child>
+            <container:child layoutData="{layoutData}">
+                <d:ToolImplementationEditor ui:field="implementationEditor"/>
+            </container:child>
+            <container:child layoutData="{layoutData}">
+                <d:ToolContainerEditor ui:field="containerEditor"/>
+            </container:child>
+        </container:VerticalLayoutContainer>
+    </g:FormPanel>
+</ui:UiBinder> 
```

Comment:
Good point. So far that directory only contains the minimal required to import the [iPlant UI Layer Code Style.xml](https://github.com/iPlantCollaborativeOpenSource/DE/blob/dev/ui/iplant/code-style/idea/codestyles/iPlant%20UI%20Layer%20Code%20Style.xml) file, which is specific to Java code-style settings, but the `Ensure line feed at file end on Save` isn't part of those settings.
I can look into trying to include this setting in another file in that directory (maybe it's supposed to go in the [IntelliJ IDEA Global Settings](https://github.com/iPlantCollaborativeOpenSource/DE/blob/dev/ui/iplant/code-style/idea/IntelliJ%20IDEA%20Global%20Settings) file).